it/uk/gov/hmrc/tai/integration/UpdateWorkingFromHomeExpensesSpec.scala

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmrc.tai.integration
+
+class UpdateWorkingFromHomeExpensesSpec {
```

Comment:
Is this still an empty class?

Reply:
Yeah we need IT tests for this endpoint as its a really busy one thats untested in our ATs as its used by another service.